### PR TITLE
Fix links, redirects

### PR DESCRIPTION
**Pull request by mirii1994**
In this PR:
- Fix bug for double `//` on links.
- Fix bug for redirects - each redirect will be sent as a separate log.
- Update dependency with vulnerability.
- Match go version in go releaser to go version in go.mod (1.19).
- Update reamde - changelog, links.

### CODE REVIEW

File: main.go [hunk 1]
```diff
@@ -359,7 +394,20 @@ func getLinksInPage(url string) []string {
 					link := a.Val
 					matched, _ := regexp.MatchString("^http(s?)://", link)
 					if !matched {
-						link = url + link
+						tmpUrl := url
```
Comment (Doron-Bargo) [inline main.go:397]: Looking good
Can we move it to a new function - maybe "clean link url" or something similar
Also write some comments on this process to make it more clear

**Review by Doron-Bargo — CHANGES_REQUESTED** [review]
Please check my comments

File: main.go (hunk 1 above)
Comment (mirii1994) [reply main.go]: Done.

**Review by Doron-Bargo — APPROVED** [review]
Great work

**Pull request merged**